src/test/java/set/SetTest.java

```diff
@@ -0,0 +1,46 @@
+package set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+
+	@DisplayName("Set의 크기를 확인한다")
+	@Test
+	void size() {
+		assertThat(numbers.size()).isEqualTo(3);
```

Comment:
```suggestion
		assertThat(numbers).hasSize(3);
```
이렇게도 해볼 수 있을 것 같아요 좀 더 사이즈를 검증하는게 명확해질 수 있을 것 같습니다